### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by lhorie**
- re-add tests for install / add / upgrade / remove
- add new test for bool_shouldinstall
- fix upgrade edge case

### CODE REVIEW

File: commands/upgrade.js [hunk 1]
```diff
@@ -66,8 +66,8 @@ const upgrade /*: Upgrade */ = async ({root, args}) => {
 
 const update = (meta, type, name, version, from) => {
   if (meta[type] && meta[type][name]) {
-    const min = minVersion(meta[type][name]);
```
Comment (lhorie) [inline commands/upgrade.js:69]: this would throw if upgrading and the existing version is not valid semver, e.g. `workspace:*`

File: tests/index.js [hunk 2]
```diff
@@ -133,7 +133,7 @@ async function runTests() {
   // run separately to avoid CI error
   await t(testBazelDummy);
   await t(testBazelBuild);
-  // await t(testInstallAddUpgradeRemove);
+  await t(testInstallAddUpgradeRemove);
```
Comment (lhorie) [inline tests/index.js:136]: re-enable this test since it's a pretty important one

**Review by chrisdothtml — APPROVED** [review]
